### ISSUE TITLE
Consider different name for PostBack.Update. 

### ISSUE THREAD

**Issue by quigamdev** [riganti/dotvvm#202]
When component contains `PostBack.Update` which is true, it means the components html is refreshed. 
By my opinion, `PostBack.Update` is confusing name. What about something like `RenderSettings.PostBackUpdatesHtml` ?

**Comment by exyi** [comment]
The control 'updates on postback'. And the property should not be in the class `RenderSettings` because it does not control how the control is rendered but how it behaves on postback. I don’t think we should rename it. It is also pretty heavily used and everyone would have to rewrite his code or we’d have to write automatic rewriter.

**Comment by tomasherceg** [comment]
Actually, I also don't think `PostBack.Update` is a good name. I took the `Update` word from the Web Forms `UpdatePanel` control, and instead of creating a control, I have implemented it as the attached property.

I don't think it is used on many places so we could rename it, however I don't like the suggested names. I think we can rename it to `PostBack.UpdateHtml` because that's what the property does - it renders the control on each postback and updates the HTML in the page DOM.
